cos-fleet-catalog-connectors/azure/azure-eventhubs-0.1/pom.xml

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+
+    <parent>
+        <groupId>org.bf2</groupId>
+        <artifactId>cos-fleet-catalog-connectors-azure</artifactId>
+        <version>999-SNAPSHOT</version>
+    </parent>
+
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>cos-connector-azure-eventhubs-0.1</artifactId>
+
+    <name>Azure Event Hubs</name>
+    <description>Azure Event Hubs</description>
+
+    <packaging>jar</packaging>
+
+    <properties>
+        <cos.connector.type>azure-eventhubs</cos.connector.type>
+        <quarkus.container-image.name>cos-connector-${cos.connector.type}</quarkus.container-image.name>
+        <quarkus.container-image.tag>${cos.connector.version}.${cos.connector.revision}</quarkus.container-image.tag>
+    </properties>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.bf2</groupId>
+                <artifactId>camel-connector-maven-plugin</artifactId>
+                <configuration>
+                    <connectors>
+                        <connector>
+                            <name>${cos.connector.type}-source-${cos.connector.version}</name>
+                            <title>${project.name} source</title>
+                            <description>${project.description} source</description>
+                            <adapter>
+                                <prefix>azure</prefix>
+                                <name>${cos.connector.type}-source</name>
+                                <version>${camel-kamelets-catalog.version}</version>
+                            </adapter>
+                            <kafka>
+                                <prefix>kafka</prefix>
+                                <name>cos-kafka-sink</name>
+                                <version>${cos.kamelets.version}</version>
+                            </kafka>
+                            <dataShape>
+                                <produces>
+                                    <formats>
+                                        <format>application/octet-stream</format>
```

Comment:
the kamelet produces json https://github.com/apache/camel-kamelets/blob/main/kamelets/azure-eventhubs-source.kamelet.yaml#L84-L85